# Single-cell gene regulatory network analysis reveals potential mechanisms of action of antimalarials against SARS-CoV-2

> James J. Cai<sup>1,2,3,*</sup>, Daniel Osorio<sup>1</sup>
> <sup>1</sup>Department of Veterinary Integrative Biosciences, <sup>2</sup>Department of Electrical and Computer Engineering, <sup>3</sup>Interdisciplinary Program of Genetics, Texas A&M University, College Station, TX 77843, USA.
> ORCIDs: 0000-0002-8081-6725 (J.J.C.), 0000-0003-4424-8422 (D.O.)
> <sup>*</sup>Corresponding author: J.J.C. (jcai@tamu.edu)

**Abstract**
The efficiency of antimalarials, chloroquine (CQ) and hydroxychloroquine (HCQ), in the prevention and treatment of coronavirus disease 2019 (COVID-19) is under intense debate. The mechanisms of action of antimalarials against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) have not been fully elucidated. Here, we applied a network-based comparative analysis, implemented in our machine learning workflowâ€”scTenifoldNet, to scRNA-seq data from COVID-19 patients with different levels of severity. We found that genes of the Malaria pathway expressed in macrophages are significantly differentially regulated between patients with moderate and severe symptoms. Our findings help reveal the mechanisms of action of CQ and HCQ during SARS-CoV-2 infection, providing new evidence to support the use of these antimalarial drugs in the treatment of COVID-19, especially for patients who are mildly affected or in the early stage of the infection.

[*OSF Preprints*. 10.31219/osf.io/va7ux](https://doi.org/10.31219/osf.io/va7ux)
